main/templates/main/snippets/navbar.html

```diff
@@ -66,6 +66,15 @@
                     <li class="nav-item {% if url_name == 'about' %}active{% endif %}">
                         <a class="nav-link" href="{% url 'about' %}">About</a>
                     </li>
+                    <li class="nav-item {% if url_name == 'self-assess' %}active{% endif %}">
+                        <a class="nav-link" href="{% url 'self-assess' %}">Self Assessment Questionnaire</a>
```

Comment:
Should I shorten this name as well?
```suggestion
                        <a class="nav-link" href="{% url 'self-assess' %}">Self Assessment</a>
```